components/hierarchical-view/hierarchical-view-mixin.js

```diff
@@ -113,6 +101,18 @@ export const HierarchicalViewMixin = superclass => class extends superclass {
 		window.removeEventListener('resize', this.__onWindowResize);
 	}
 
+	firstUpdated() {
+		super.firstUpdated();
+
+		this.addEventListener('keyup', this.__onKeyUp);
+		this.addEventListener('d2l-hierarchical-view-hide-start', this.__onHideStart);
+		this.addEventListener('d2l-hierarchical-view-show-start', this.__onShowStart);
+		this.addEventListener('d2l-hierarchical-view-resize', this.__onViewResize);
+		this.__onWindowResize = this.__onWindowResize.bind(this);
```

Comment:
I think this bind should be done in the constructor.  Currently, the handler is wired up in `connectedCallback` (which I think is good), but this line will run afterwards.

Reply:
I think this still need to get moved up.